ocf_datapipes/batch/convert_to_xarray_batch.py

```diff
@@ -0,0 +1,20 @@
+from typing import Iterable, Union
+
+import xarray as xr
+from torchdata.datapipes import functional_datapipe
+from torchdata.datapipes.iter import IterDataPipe, Zipper
+
+from ocf_datapipes.utils.consts import XarrayBatch
+
+
+@functional_datapipe("convert_to_xarray_batch")
+class ConvertToXarrayBatchIterDataPipe(IterDataPipe):
+    def __init__(self, source_dps: Iterable[IterDataPipe]):
+        super().__init__()
+        self.source_dps = source_dps
```

Comment:
can we change dp=datapipe, i would prefer not to have abbreivates, if we dont need them

Reply:
Yep, I'll refactor it for that tomorrow